lib/mquery.js

```diff
@@ -1402,12 +1402,18 @@ function _pushArr (opts, field, value) {
  * limit, skip, maxScan, batchSize, comment
  *
  * Sets these associated options.
+ * With mongodb>=3.2.6 if the value is a string doesn't accept it, converting type string numbers to numeric type in applicable cases
  *
  *     query.comment('feed query');
  */
 
 ;['limit', 'skip', 'maxScan', 'batchSize', 'comment'].forEach(function (method) {
   Query.prototype[method] = function (v) {
+    if (method !== 'comment') {
+      let value = Number(v);
```

Comment:
This lib still supports node 0.10 and 0.12, so can't use `let` or `Number.isNaN`.